packages/hub/src/users.spec.ts

```diff
@@ -2,18 +2,21 @@ import path from 'path'
 import fs from 'fs'
 import { expect } from 'chai'
 import { isBrowser } from 'browser-or-node'
+import { grpc } from '@improbable-eng/grpc-web'
```

Comment:
is this needed? I don't see its usage below, but I might just be missing something?